README.md

```diff
@@ -194,4 +192,4 @@ I would like to thank Microverse<br>
 
 This project is [licensed](./LICENSE) .
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
\ No newline at end of file
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- Kindly fix the broken License as suggested by the previous reviewer 😉
![Screenshot from 2023-01-20 22-34-31](https://user-images.githubusercontent.com/44624138/213809671-c80b4007-1563-484e-9b1a-ea8927abac36.png)